src/commands/setup-docker.yml

```diff
@@ -3,20 +3,21 @@ description: >
   This command should be used prior to running any docker commands that interact with the remote repository.
 
 parameters:
-  docker-username:
+  docker-password:
+    default: DOCKER_PASSWORD
+    description: |
+      Name of environment variable storing your Docker password
     type: env_var_name
+  docker-username:
     default: DOCKER_USERNAME
-    description: >
-      Username for the docker acccount
-
-  docker-password:
+    description: |
+      Name of environment variable storing your Docker username
     type: env_var_name
-    default: DOCKER_PASSWORD
-    description: >
-      Password for the docker account
 
 steps:
   - setup_remote_docker
   - docker/install-docker
-  - run: |
-    docker login -u << parameters.docker-username >> -p << parameters.docker-password >>
\ No newline at end of file
+  - run:
```

Comment:
You can use docker/check (it is a command in the circleci orb. 
<img width="495" alt="Screen Shot 2022-04-14 at 2 54 53 PM" src="https://user-images.githubusercontent.com/70287489/163457816-d3906f1d-9ca6-4be2-ab9c-96b3844acea9.png">